blogposts/2020/3.18/C++-precise-code.md

```diff
@@ -0,0 +1,23 @@
+
+---
+title: 'New C++ precise code intelligence solution'
+author: 'Garo Brik'
+publishDate: 2020-07-20T00:00-07:00
+tags: ["blog"]
+slug: indexed-non-master-branches
+
+---
```

Comment:
I think you're missing two metadata items here:
```
heroImage: https://about.sourcegraph.com/sourcegraph-mark.png
published: true
```

Reply:
<3